content/en/docs/setup/production-environment/windows/user-guide-windows-containers.md

```diff
@@ -177,7 +177,7 @@ This label reflects the Windows major, minor, and build number that need to matc
 1. Save this file to `runtimeClasses.yml`. It includes the appropriate `nodeSelector` for the Windows OS, architecture, and version.
 
 ```yaml
-apiVersion: node.k8s.io/v1beta1
+apiVersion: node.k8s.io/v1
```

Comment:
there are many occurences of this in other language docs. Should I include those in this PR?

Reply:
pinging @sftim @kbhawkey @tengqm who can help answer this question

Reply:
Hi. Typically, you should limit pull request changes to one language and let the localization teams update the
corresponding content.
I am unaware of a formal process for localization teams to follow-up on updates to the `dev-*` branch.
If the changes are straightforward, it would be a possible to get review/approval from a localization team before approving this PR.

Reply:
I can send a separate PR than once this it merged

Reply:
_several_ separate PR's I'm afraid - each language has its own reviewers. Most of us can't review other localizations although we have a few polyglots.